src/main/java/org/dcsa/bkg/model/transferobjects/ShipmentLocationTO.java

```diff
@@ -8,19 +8,27 @@
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Size;
 import java.time.OffsetDateTime;
+import java.util.UUID;
 
 @Data
 public class ShipmentLocationTO {
+  @NotNull(message = "ShipmentID is required.")
+  private UUID shipmentID;
+
+  @NotNull(message = "BookingID is required.")
+  private UUID bookingID;
```

Comment:
Not sure if `bookingID` is required here, I don't see it in the `shipmentLocation` object on swagger.